cannot do npm install on fresh install

Log truncated for relevant parts.

```
➜  ~PERSONA_ROOT git:(dev) ✗ npm install

> browserid@1.0.0-b1 preinstall /Users/stomlinson/development/browserid
> node ./scripts/lockdown

NPM Lockdown is here to check your dependencies!  Never fear!

> browserid@1.0.0-b1 preinstall /Users/stomlinson/development/browserid
> node ./scripts/lockdown
...

npm http GET http://127.0.0.1:54482/connect-fonts/0.0.9-alpha8
npm http GET https://github.com/downloads/lloyd/node-statsd/0509f85.tgz
npm http 200 http://127.0.0.1:54482/connect-fonts/0.0.9-alpha8
npm http GET http://127.0.0.1:54482/connect-fonts/-/connect-fonts-0.0.9-alpha8.tgz
npm http 200 http://127.0.0.1:54482/connect-fonts/-/connect-fonts-0.0.9-alpha8.tgz
npm http 200 https://github.com/downloads/lloyd/node-statsd/0509f85.tgz
npm http GET http://127.0.0.1:54482/node-font-face-generator/0.0.10
npm http GET http://127.0.0.1:54482/mime/1.2.9
npm http GET http://127.0.0.1:54482/send/0.1.0
npm http 200 http://127.0.0.1:54482/send/0.1.0
npm http 200 http://127.0.0.1:54482/node-font-face-generator/0.0.10
npm http GET http://127.0.0.1:54482/send/-/send-0.1.0.tgz
npm http GET http://127.0.0.1:54482/node-font-face-generator/-/node-font-face-generator-0.0.10.tgz
npm http 200 http://127.0.0.1:54482/mime/1.2.9
npm http GET http://127.0.0.1:54482/mime/-/mime-1.2.9.tgz
npm http 200 http://127.0.0.1:54482/send/-/send-0.1.0.tgz
npm http 200 http://127.0.0.1:54482/mime/-/mime-1.2.9.tgz
npm http 200 http://127.0.0.1:54482/node-font-face-generator/-/node-font-face-generator-0.0.10.tgz
npm http GET http://127.0.0.1:54482/debug
npm http GET http://127.0.0.1:54482/mime/1.2.6
npm http GET http://127.0.0.1:54482/fresh/0.1.0
npm http GET http://127.0.0.1:54482/range-parser/0.0.4
npm http GET http://127.0.0.1:54482/ua-parser/0.2.4
npm http 200 http://127.0.0.1:54482/mime/1.2.6
npm http 200 http://127.0.0.1:54482/range-parser/0.0.4
npm http 200 http://127.0.0.1:54482/fresh/0.1.0
npm http 200 http://127.0.0.1:54482/ua-parser/0.2.4
npm http GET http://127.0.0.1:54482/mime/-/mime-1.2.6.tgz
npm http GET http://127.0.0.1:54482/range-parser/-/range-parser-0.0.4.tgz
npm http GET http://127.0.0.1:54482/fresh/-/fresh-0.1.0.tgz
npm http GET http://127.0.0.1:54482/ua-parser/-/ua-parser-0.2.4.tgz
npm http 200 http://127.0.0.1:54482/debug
npm http 200 http://127.0.0.1:54482/mime/-/mime-1.2.6.tgz
npm http 200 http://127.0.0.1:54482/fresh/-/fresh-0.1.0.tgz
npm http 200 http://127.0.0.1:54482/range-parser/-/range-parser-0.0.4.tgz
npm http 200 http://127.0.0.1:54482/ua-parser/-/ua-parser-0.2.4.tgz
npm http GET http://127.0.0.1:54482/yamlparser
npm http 200 http://127.0.0.1:54482/yamlparser
npm ERR! Error: No compatible version found: yamlparser@'>=0.0.2'
npm ERR! No valid targets found.
npm ERR! Perhaps not compatible with your version of node?
npm ERR!     at installTargetsError (/Users/stomlinson/development/nvm/v0.8.12/lib/node_modules/npm/lib/cache.js:553:10)
npm ERR!     at next (/Users/stomlinson/development/nvm/v0.8.12/lib/node_modules/npm/lib/cache.js:532:17)
npm ERR!     at /Users/stomlinson/development/nvm/v0.8.12/lib/node_modules/npm/lib/cache.js:512:5
npm ERR!     at saved (/Users/stomlinson/development/nvm/v0.8.12/lib/node_modules/npm/node_modules/npm-registry-client/lib/get.js:138:7)
npm ERR!     at /Users/stomlinson/development/nvm/v0.8.12/lib/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:218:7
npm ERR!     at Object.oncomplete (fs.js:297:15)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Darwin 12.3.0
npm ERR! command "/Users/stomlinson/development/nvm/v0.8.12/bin/node" "/Users/stomlinson/development/nvm/v0.8.12/bin/npm" "install"
npm ERR! cwd /Users/stomlinson/development/browserid
npm ERR! node -v v0.8.12
npm ERR! npm -v 1.1.63
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/stomlinson/development/browserid/npm-debug.log
npm ERR! not ok code 0
npm ERR! browserid@1.0.0-b1 preinstall: `node ./scripts/lockdown`
npm ERR! `sh "-c" "node ./scripts/lockdown"` failed with 1
npm ERR!
npm ERR! Failed at the browserid@1.0.0-b1 preinstall script.
npm ERR! This is most likely a problem with the browserid package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./scripts/lockdown
npm ERR! You can get their info via:
npm ERR!     npm owner ls browserid
npm ERR! There is likely additional logging output above.

npm ERR! System Darwin 12.3.0
npm ERR! command "/Users/stomlinson/development/nvm/v0.8.12/bin/node" "/Users/stomlinson/development/nvm/v0.8.12/bin/npm" "install"
npm ERR! cwd /Users/stomlinson/development/browserid
npm ERR! node -v v0.8.12
npm ERR! npm -v 1.1.63
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/stomlinson/development/browserid/npm-debug.log
npm ERR! not ok code 0
➜  ~PERSONA_ROOT git:(dev) ✗
```

The author of yamlparser force pushed a new 0.0.2 version this morning, causing a sha mismatch:

```
➜  ~PERSONA_ROOT git:(dev) ✗ npm show yamlparser

{ name: 'yamlparser',
  description: 'A YAML parser written in javascript. This is the port to CommonJS. Client-side script is available at http://code.google.com/p/javascript-yaml-parser/',
  'dist-tags': { latest: '0.0.2' },
  versions: [ '0.0.1', '0.0.2' ],
  maintainers: 'h4evr <costa.h4evr@gmail.com>',
  time:
   { '0.0.1': '2011-08-08T22:59:44.100Z',
     '0.0.2': '2013-05-01T10:11:19.706Z' },
  author: 'Diogo Costa <costa.h4evr@gmail.com> (http://diogocosta.pt.tl/)',
  repository:
   { type: 'git',
     url: 'git://github.com/h4evr/commonjs-javascript-yaml-parser.git' },
  users: { substack: true, pid: true },
  version: '0.0.2',
  main: 'yamlparser',
  readmeFilename: 'README.markdown',
  dist:
   { shasum: '32393e6afc70c8ca066b6650ac6738b481678ebc',
     tarball: 'http://registry.npmjs.org/yamlparser/-/yamlparser-0.0.2.tgz' },
  directories: {} }

➜  ~PERSONA_ROOT git:(dev) ✗
```

--- a/lockdown.json
+++ b/lockdown.json
@@ -381,7 +381,7 @@
     "0.1.3": "4448cb94fb1032c02cb2c81e95008674dcd2d1c6"
   },
   "yamlparser": {
-    "0.0.2": "e9f3e05e52762df751e4c522e0b6a43fff9ce1c4"
+    "0.0.2": "32393e6afc70c8ca066b6650ac6738b481678ebc"
   },
   "zeparser": {
     "0.0.5": "03726561bc268f2e5444f54c665b7fd4a8c029e2"

